docs/app/Examples/elements/List/Types/index.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import { Message } from 'stardust'
+
+import ComponentExample from 'docs/app/Components/ComponentDoc/ComponentExample'
+import ExampleSection from 'docs/app/Components/ComponentDoc/ExampleSection'
+
+const ListTypes = () => (
+  <ExampleSection title='Types'>
+    <ComponentExample
+      title='List'
+      description='A list groups related content'
+      examplePath='elements/List/Types/ListExampleBasic'
+    />
+    <ComponentExample examplePath='elements/List/Types/ListExampleIcon' />
+    <ComponentExample examplePath='elements/List/Types/ListExampleDivided' />
+    <ComponentExample examplePath='elements/List/Types/ListExampleTree' />
+
+    <ComponentExample
+      title='Bulleted'
+      description='A list can mark items with a bullet'
+      examplePath='elements/List/Types/ListExampleBulleted'
+    />
+    <ComponentExample examplePath='elements/List/Types/ListExampleBulletedSimple'>
+      <Message info>
+        For convenience, a simple bulleted list can also use <code>ul</code> tag.
```

Comment:
I think this message is only applicable to SUI core where the user defines HTML/CSS.  Since, in Semantic-UI-React the classNames are hidden from the user.  Let's go ahead and remove this.